Source/WebCore/platform/graphics/cocoa/IOSurface.h

```diff
@@ -122,17 +122,17 @@ class IOSurface final {
 
     // Any images created from a surface need to be released before releasing
     // the surface, or an expensive GPU readback can result.
```

Comment:
It's releasing the `CGContext` that will trigger read back of any `CGImages` created using it, right? It was previously this surface, since it was (hopefully, maybe) the only one holding a ref to the `CGContext`.